docs/specs.md

```diff
@@ -82,12 +82,48 @@ We need to balance two conflicting requirements
 
 There are quite sophisticated “Layout” formats for text documents: Word documents (Office Open XML aka OOXML), PDF files, RTF, ODF (Open Document), etc. They are very complex, with specifications compressing many pages, since they allow the complete and precise specification of all aspects of document layout, structure and presentation. They would, of course, offer the greatest nuance in determining the relationships between the text chunks they contain[^1], but would be too difficult to handle for our purpose of consolidating a data interchange format for PII processing that has “reasonable” complexity.
 
-Instead, we are aiming at a simpler solution. As a very minimum, a document can be considered as _a collection of text chunks_. How those chunks are structured is what generates the model to be represented. In general terms, we will consider two main document models:
+Instead, we are aiming at a simpler solution. As a very minimum, a document
+can be considered as _a collection of text chunks_. How those chunks are
+structured is what generates the model to be represented. In general terms, we
+will consider three main document model types:
 
-* a **hierarchical** model: a top-down structure relates text chunks one to another
+* a **sequential** model: a document is composed by a list of consecutive chunks
+* a **tree** model: a hierarchical top-down structure relates text chunks one to another, so that chunks can be nested
```

Comment:
well, not necessarily, since a JSON file might contain _anything_: a tree, a list, a flat dictionary, etc. 

A tree is meant to represent a typical structured document, having sections, subsections, subsubsections, etc. The source format might be Word, HTML, PDF, etc; any file format capable of containing something akin to "headings". How to translate those headings & paragraphs into a tree would be the topic for _another_ package, since the pii-data package should be file-format-agnostic, I believe

Reply:
I thought nested chunks fit well with more complex formats, such as EML files (tree structure of emails + attachments) or Excel files (multiple sheet tabs = multiple tables in a single file).

Reply:
That might well be. A tree-based format is flexible. Though an Excel with tabs as a nested file would need a mixed format tree+tabular.

This is possible and interesting, although I would not complicate it that much in the first version, to make it more manageable.